spark-doris-connector/src/main/java/org/apache/doris/spark/load/DorisStreamLoad.java

```diff
@@ -408,7 +411,10 @@ public Map<String, String> getStreamLoadProp(SparkSettings sparkSettings) {
 
     private String getBackend() {
         try {
-            if (autoRedirect) {
+            /**
+             * It says that when BE exists there is no FE so this is handled logically
+             */
+            if (autoRedirect && null == beNodes) {
                 return RestService.randomEndpoint(fenodes, LOG);
```

Comment:
Is it possible to use autoRedirect directly to judge here? There is no need to add benodes=null logic, otherwise the configuration will be confusing.

Reply:
ok  let me think about this one